Criação do README.md

juant-tavares opened the pull request:
Criei o README baseado nas informações básicas da descrição do whats e no que vi durante minhas contribuições até agora, sem ir a fundo na questão da estrutura e não possuindo todas as informações como nome completo dos contribuintes.

juant-tavares committed:
Create README.md

Criei o README baseado nas informações básicas da descrição do whats e no que vi durante minhas contribuições até agora, sem ir a fundo na questão da estrutura e não possuindo todas as informações como nome completo dos contribuintes.

FranciellyDiasM commented on README.md:
```diff
@@ -0,0 +1,41 @@
+
+  Tecnologias Utilizadas
+
+Kotlin: Linguagem de programação utilizada para o desenvolvimento do aplicativo Android.
+SQLite: Banco de dados local para armazenar informações sobre os usuários e cardápios personalizados.
+GitHub: Repositório para controle de versão e colaboração entre os membros da equipe.
```
Muito legal essa parte! o que acha de acrescentarmos sobre que temos um board kanbam no trello para gerenciar o projeto?

juant-tavares replied:
Muito obrigado pela dica! Adicionei lá no campo de tecnologias.

juant-tavares committed:
Atualização dos colaboradores

Adicionei nome completo dos integrantes do projeto e retirei aqueles que não faziam parte da disciplina de Banco de Dados.

juant-tavares committed:
Adição do Trello como tecnologia

Seguindo o comentário da Francielly, adicionei à lista de ferramentas usadas no projeto, o Kanban.
